components/BottomAppBar/src/ColorThemer/MDCBottomAppBarColorThemer.h

```diff
@@ -14,10 +14,10 @@
  limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
+#import "MaterialBottomAppBar.h"
+#import "MaterialColorScheme.h"
 
-@class MDCBottomAppBarView;
```

Comment:
Are we discouraging forward declarations?

Reply:
Of APIs outside of a given target, yes.